packages/material-ui/src/Tabs/Tabs.d.ts

```diff
@@ -10,9 +9,10 @@ declare const Tabs: OverridableComponent<{
     children?: React.ReactNode;
     indicatorColor?: 'secondary' | 'primary' | string;
     onChange?: (event: React.ChangeEvent<{}>, value: any) => void;
+    orientation?: 'horizontal' | 'vertical';
     ScrollButtonComponent?: React.ElementType;
     scrollButtons?: 'auto' | 'desktop' | 'on' | 'off';
-    TabIndicatorProps?: Partial<TabIndicatorProps>;
+    TabIndicatorProps?: Partial<React.HTMLAttributes<HTMLDivElement>>;
```

Comment:
Not disagreeing with this change but we should update the documentation as well: 

> Properties applied to the TabIndicator element.

Since this isn't a documented component we shouldn't use a proper noun here.